tests/examples/step-34.diff

```diff
@@ -0,0 +1,43 @@
+497c497
+<           in.open("coarse_circle.inp");
+---
+>           in.open("../../../source/step-34/coarse_circle.inp");
+501c501
+<           in.open("coarse_sphere.inp");
+---
+>           in.open("../../../source/step-34/coarse_sphere.inp");
+974,983c974,983
+<     DataOut<dim> data_out;
+< 
+<     data_out.attach_dof_handler(external_dh);
+<     data_out.add_data_vector(external_phi, "external_phi");
+<     data_out.build_patches();
+< 
+<     const std::string filename = std::to_string(dim) + "d_external.vtk";
+<     std::ofstream     file(filename);
+< 
+<     data_out.write_vtk(file);
```

Comment:
This is the only line you would need to comment out....
But, at the same time, do we actually need to disable graphical output? There is an advantage to executing this code. (I don't feel strongly about it because one could argue that this would speed up things)